apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -576,6 +586,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
       plugins: {
         blockDragger: ScrollBlockDragger,
         metricsManager: CdoMetricsManager,
+        connectionChecker: CdoConnectionChecker,
```

Comment:
I see that `cdoConnectionChecker` is registered above in line 274. Are we required to add it as a plugin even after it's registered?

Reply:
Okay. That make sense.
Also, just including a link to [`options.ts`](https://github.com/google/blockly/blob/develop/core/options.ts) which lists the Blockly options. Could you confirm that since `connectionChecker` is not in this list, we add as a plugin. But since `theme` is in the list, we don't need to add any of the customized themes as a plugin. Thanks!

Reply:
Yes, that is correct and it was also the path recommended by the Blockly team. 

This page documents the best practices for replacing certain classes, including `connectionChecker`: https://developers.google.com/blockly/guides/configure/advanced/using_blockly_apis

Reply:
One thing I've realized is that we don't strictly need to register the class in order for it to work when we pass it directly into the inject, however it's still beneficial to add it to the registry. For example, if we wanted different labs to use different connection checkers, we'd be able to specify them as part of `options` just by using a string like `'cdo_connection_checker'` rather than having to import the entire class in whichever files set the inject options.

Reply:
Thanks for the explanation and documentation!